plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -310,7 +312,8 @@ public DeltaLakeMetadata(
             boolean deleteSchemaLocationsFallback,
             DeltaLakeRedirectionsProvider deltaLakeRedirectionsProvider,
             ExtendedStatisticsAccess statisticsAccess,
-            boolean useUniqueTableLocation)
+            boolean useUniqueTableLocation,
+            MetastoreTypeConfig metastoreTypeConfig)
```

Comment:
`String metastoreType` is better as we should accept only required field (the entire MetastoreTypeConfig is unnecessary).